README.md

```diff
@@ -19,56 +18,116 @@ Requirements
-```
-  * Bind address for docker enviroments
-```bash
-$ ./bin/console toolforge:ssh --bind-address=0.0.0.0
-```
+   ```console
+   symfony server:start -d
+   ```
+
+4. Open your web browser to e.g. <http://localhost:8000>
+   and the basic operations should be working with the following requirements:
+   * you can only export to EPUB format; and
+   * you must check the 'Exclude credits' option,
+     to avoid querying the database for usernames.
+
+   Continue on for setting up more exporting and development options.
+
+5. Install optional dependencies:
+
+   **Calibre:**
+   In order to export to PDF, plain text, RTF, or Mobi formats
+   you must install [Calibre](https://calibre-ebook.com)
+   so that the tool can use the `ebook-convert` command.
+
+   **epubcheck:**
+   To validate exported ebooks (with the `./bin/console app:check` command),
+   you must install [epubcheck](https://github.com/w3c/epubcheck).
+   If it's not installed at `/usr/bin/epubcheck` then
+   set the `EPUBCHECK_JAR` environment variable.
+
+   **Fonts:**
+   To run the integration tests, also install
+   the `fonts-linuxlibertine` package.
+   You can also install any other fonts you want to use for exporting books.
+
+6. WS Export uses two database connections:
+   firstly, to its own database to store download statistics;
+   and secondly, it connects to the Wikimedia database replicas
+   to fetch information about Wikisource contributors
+   for the credits list that can be included at the end of exported books.
+
+   For the statistics recording database you create the the database
+   and add connection details to `.env.local`:
+
+   ```console
+   DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name
+   ```
+
+7. Then create the database tables with:
+
+   ```
+   ./bin/console doctrine:migrations:migrate`
+   ```
+
+8. To connect to the Wikimedia replicas, this tool uses
+   the [Toolforge Bundle](https://github.com/wikimedia/ToolforgeBundle),
+   and connects to [multiple databases](https://github.com/wikimedia/ToolforgeBundle#replicas-connection-manager).
+
+   Set replicas credentials in your `.env.local` file according to the Bundle's documentation.
+
+9. Establish an SSH tunnel to the replicas:
+
+  ```console
+  ./bin/console toolforge:ssh
+  ```
+
+10. At this point, you should be able to use all of the functionality
+    of WS Export both via the web interface and CLI.
 
 ## CLI Usage
 
 ### app:check
 
-Run epubcheck on books. With no options set, this will check 10 random books from English Wikisource. Note that the random 10 will be cached (for repeatability) unless you use <info>--nocache</info>.
+Run epubcheck on books. With no options set, this will check 10 random books from English Wikisource.
+Note that the random 10 will be cached (for repeatability) unless you use <info>--nocache</info>.
 
-    app:check [-l|--lang LANG] [--nocache] [-t|--title TITLE] [-c|--count COUNT] [-s|--namespaces NAMESPACES]
+```console
```

Comment:
@waldyrious I didn't notice this before, sorry. This section of CLI usage is auto-generated (by [this thing](https://packagist.org/packages/samwilson/console-readme-generator)), so shouldn't be changed by hand. I thought we had a CI check in place to guard against this; I'm adding such a thing in #476.

Reply:
Sure, no worries. Would you like me to revert the changes to the "CLI Usage" section, and keep the changes to the other sections?

Reply:
In the meantime, I submitted https://github.com/samwilson/console-readme-generator/pull/7 to change this formatting in the source.

Reply:
Thanks! And no, it's fine to leave them as-is here, I'll merge that change and then we can update the package here.